docs/02-app/02-api-reference/02-file-conventions/01-metadata/opengraph-image.mdx

```diff
@@ -394,7 +394,7 @@ export default async function Image({ params }) {
 
 #### Using Edge runtime with local assets
 
-This example uses the Edge runtime to fetch a local image on the file system and passes it as an `ArrayBuffer` to the `src` attribute of an `<img>` element.
+This example uses the Edge runtime to fetch a local image on the file system and passes it as an `ArrayBuffer` to the `src` attribute of an `<img>` element. The local asset should be placed relative to the root of your project, instead of the example source code location.
```

Comment:
```suggestion
This example uses the Edge runtime to fetch a local image on the file system and passes it as an `ArrayBuffer` to the `src` attribute of an `<img>` element. The local asset should be placed relative to the example source file location.
```

When using `import.meta.url` like this it is relative to the source file.

Reply:
Thanks for pointing out, I did not test edge runtime and just made an assumption. I think I'll probably leave edge runtime part untouched and make another PR if I have bandwidth to verify

I'll change that later today

Reply:
oh just notice your suggested change, let me apply it